### PR TITLE
Development

### PR DESCRIPTION
**Pull request by moudio**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                    
+
+                    </section>
+
+
+            </div>
+
+            <div class="em-row html-css">
+                <div class="em-col-12">
+                    <div class="html-block">
```
Comment (maelfosso) [inline index.html]: Please, use `<section>` and `<article>` here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                        <h3 class="html-info">PROJECT: DESIGN YOUR OWN GRID-BASED FRAMEWORK</h3>
+                        <div class="html-block-buttons">
+                                <a href="#" class="view-button">View Course</a>
+                                <a href="#" class="login-button">Login to track progress</a>
+                                <a href="#" class="lesson-button">Next Lesson</a>
+                        </div>
+
+                    </div>
+
+                    <div class="question">
```
Comment (maelfosso) [inline index.html]: Please, use `<section>` or `<article>` here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                         <p class="question-heading">Have a question?</p>
+                         <p class="question-info">Chat with our friendly Odin community in our Discord chatrooms!</p>
+                         <div class="get-started"><a href="#" class="discord-button">Open Discord</a></div>
+                    </div>
+
+                    <div class="accelerate">
```
Comment (maelfosso) [inline index.html]: Please, use `<section>` or `<article>` here.

**Comment by maelfosso** [comment]
Hello, 

You did a very nice job. But before moving on, please there are some changes to do.

- [ ] Please, correct errors from https://validator.w3.org

- [ ] In your **framework.css**, please, you should define breakpoint for screen from `992px` to `1200px`. There is nothing concerning that range.

- [ ] There are no classes for margin top, bottom, left, right, all margins and auto. Please, define them. Same thing for padding.

Please, make those changes and fill a new code review request.

**Comment by Torres-ssf** [comment]
### Great work @MouhaDiouf @emmanuelkamala 
I'll give this project a 👍. You can merge to master now. Good luck with the next project.

**Pull request merged**